src/routes/TimelineLink.svelte

```diff
@@ -1,4 +1,8 @@
-<a class="panel" href="/timeline">
+<script lang="ts">
+	import {base} from '$app/paths';
+</script>
+
+<a class="panel" href="{base}/timeline">
 	<div class="icon">🗺</div>
```

Comment:
```suggestion
	<div class="icon">⏳</div>
```
Since we'd been discussing this on Felt